app/views/shared/_item_list.html.erb

```diff
@@ -0,0 +1,31 @@
+<div class="row">
+  <div class="col-md-12" id = "title">
+    <h2>All Items</h2>
+  </div>
+</div>
+
+<% @items.each_with_index do |item, index| %>
+  <% if index % 3 == 0 && index != 0 %>
```

Comment:
hmm, maybe we can state this is a more straightforward way; i assume this is to make the right number of items show up in the columns in our grid?